music_server.py

```diff
@@ -52,6 +54,25 @@ def specific_album():
 	# Return the list of tracks on the specified album
 	return json.dumps(specific_album)
 
+
+SEARCH_FOLDER_PATH = r"C:\Users\davis_g7\OneDrive\Documents\Duq-CPMA-536-Spring-2024\Music"
+#SEARCH_FOLDER_PATH = r"Music"
```

Comment:
Can you replace the `SEARCH_FOLDER_PATH` value with the relative path instead? You should be able to completely remove the absolute path version of the variable, as the relative path should work in your development environment as well.